src/Controller.js

```diff
@@ -74,7 +60,7 @@ export default class Controller {
     const self = this;
     this._$links.addClass('lightbox-link').click(function(e) {
       e.stopPropagation();
-      self.open(self._slides.filter(slide => slide.$node.is(this))[0].id);
+      self.open(self.slides.filter(slide => slide.$node.is(this))[0].id);
```

Comment:
@designjockey can't; I need to use jQuery's `this` to point to the node clicked a few characters further.